shell/common/pipeline_unittests.cc

```diff
@@ -89,46 +89,34 @@ TEST(PipelineTest, PushingMultiProcessesInOrder) {
   ASSERT_EQ(consume_result_2, PipelineConsumeResult::Done);
 }
 
-TEST(PipelineTest, PushingToFrontOverridesOrder) {
+TEST(PipelineTest, ProduceIfEmptyWhenQueueIsNotEmptyDoesNotConsume) {
```

Comment:
rename test to `ProduceIfEmptyDoesNotConsumeWhenQueueIsNotEmpty`

Reply:
done